samples/azure-quantum/parallel-qrng/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+author: cgranade
+description: Generate random numbers in parallel on quantum hardware, using the Azure Quantum service
+ms.author: chgranad@microsoft.com
+ms.date: 01/25/2021
+languages:
+- qsharp
+- csharp
+- python
+products:
+- qdk
+- azure-quantum
+---
+
+# Parallel quantum random number generator (QRNG)
```

Comment:
Add "sample" or even better "example" to H1.
People search way more on "example" for various tech, even though internally to MSFT we're used to saying "sample."